src/Routes.php

```diff
@@ -246,11 +247,47 @@ protected function sliceEndpoints(): array
         return $routes;
     }
 
+    protected function actionEndpoints(): array
+    {
+        $actions = app($this->formation)->actions();
+
+        if (!count($actions)) {
+            return [];
+        }
+
+        $routes = [];
+
+        /** @var Action $action */
+        foreach ($actions as $action) {
+            $routes[] = [
+                'type'         => 'store',
+                'verb'         => 'POST',
+                'action'       => [ActionController::class, 'store'],
+                'name'         => "$this->resource.actions.{$action->key}.store",
+                'endpoint'     => "actions/$this->resource/{$action->key}",
+                'key'          => "$this->resource.actions.{$action->key}.store",
+                'with-trashed' => $this->withTrashed('store'),
```

Comment:
this will be false since there is no model (with trashed is for SoftDeletes)

Reply:
Done